### PR TITLE
updated how created and last modified fields are set

### PR DESCRIPTION
**Pull request by anthony-britton-moj**

### CODE REVIEW

File: projects/prison-case-notes-to-probation/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/audit/AuditedInteraction.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    val dateTime: ZonedDateTime = ZonedDateTime.now(),
+
+    val businessInteractionId: Long,
+    val userId: Long,
+
+    @Convert(converter = AuditedInteractionOutcomeConverter::class)
+    val outcome: Outcome = Outcome.SUCCESS,
+
+    @Column(name = "interaction_parameters", length = 500)
+    @Convert(converter = AuditedInteractionOutcomeConverter::class)
```
Comment (marcus-bcl) [inline projects/prison-case-notes-to-probation/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/audit/AuditedInteraction.kt:24]: Should this have a different converter?

File: projects/prison-case-notes-to-probation/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/audit/BusinessInteraction.kt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+@Entity(name = "business_interaction")
+class BusinessInteraction(
+    @Id @Column(name = "business_interaction_id", nullable = false)
+    private var id: Long,
+
+    @Column(name = "business_interaction_code", nullable = false)
+    private val code: String,
+
+    @Column(name = "enabled_date")
+    private val enabledDate: ZonedDateTime,
```
Comment (marcus-bcl) [inline projects/prison-case-notes-to-probation/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/audit/BusinessInteraction.kt:19]: Do we need to map this field?

**Review by marcus-bcl — APPROVED** [review]
Looks good

**Review by stevomcallister — APPROVED** [review]

File: build.gradle.kts [hunk 3]
```diff
@@ -176,7 +174,11 @@ tasks.register<JacocoReport>("jacocoTestReport") {
     dependsOn(subprojects.map { it.tasks.getByName("jacocoTestReport") })
     val main = subprojects.map { it.sourceSets.named("main").get() }
     additionalSourceDirs(files(main.map { it.allSource.srcDirs }))
-    additionalClassDirs(files(main.map { it.output }))
+    additionalClassDirs(
+        files(
+            main.map { it.output }.map { it.flatMap { file -> fileTree(file) { exclude(exclusions) } } }
+        )
+    )
```
Comment (marcus-bcl) [inline build.gradle.kts:181]: Do we need to do this?  This task just aggregates the execution data from the subprojects, so haven't the exclusions already have been applied there?

**Review by stevomcallister — APPROVED** [review]

**Pull request merged**